var/spack/repos/builtin/packages/netlib-scalapack/package.py

```diff
@@ -67,6 +67,10 @@ def scalapack_libs(self):
             'libscalapack', root=self.prefix, shared=shared, recursive=True
         )
 
+    @property
+    def libs(self):
+        return self.scalapack_libs
```

Comment:
According to the protocol for spec interface I think you just need to rename `scalapack_libs` to `libs`.

I would refer to docs for the protocol, but I think what I wrote about that in #6148 was deleted in #7144 when rewording the section. In any case you can look directly at #6148, [here](https://github.com/spack/spack/pull/6148/files#diff-d3f0f15048e7ab63c90a5e10587b28c8R229)

Reply:
You are right, I did not think of this simplification. It is clear from the rules
https://github.com/spack/spack/blob/151f305ddce7ebba78212cfd08c425bf6d9fe009/lib/spack/spack/spec.py#L785-L789
that providing just `.libs` is equivalent. I'll push the correction in a moment.